protoc_plugin/test/enum_generator_test.dart

```diff
@@ -32,14 +38,58 @@ void main() {
               ..name = 'BUSINESS'
               ..number = 2,
           ]);
+    md = DescriptorProto()..enumType.add(ed);
+  });
+
+  test('testEnumGenerator', () {
     final writer = IndentingWriter(
       generateMetadata: true,
       fileName: 'sample.proto',
     );
-    final fg = FileGenerator(FileDescriptorProto(), GenerationOptions());
+    final fg = FileGenerator(
+      testEditionDefaults,
+      FileDescriptorProto(),
+      GenerationOptions(),
+    );
     final eg = EnumGenerator.topLevel(ed, fg, <String>{}, 0);
     eg.generate(writer);
-    expectGolden(writer.emitSource(format: false), 'enum.pbenum.dart');
-    expectGolden(writer.sourceLocationInfo.toString(), 'enum.pbenum.dart.meta');
+    expectGolden(writer.emitSource(format: false), 'enum');
+    expectGolden(writer.sourceLocationInfo.toString(), 'enum.meta');
```

Comment:
Similarly (but not necessarily for this PR, as you are contributing already reviewed code), consider `enum.dart.meta`.